deployments/data-hub/data-hub--staging.yaml

```diff
@@ -0,0 +1,230 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: data-hub--staging
+  namespace: data-hub
+  annotations:
+    fluxcd.io/automated: 'true'
```

Comment:
Not important now perhaps, or perhaps not relevant at all. But this would be were you set how fluxcd should determine if a container is newer or not with some extra annotations, e.g..

```
  annotations:
    fluxcd.io/automated: "true"
    filter.fluxcd.io/airflow: glob:master-*-*
```

I can see though from the below this is quite different so might not be relevant at all, and like I say a consideration for the future so don't need to worry about it now.

Reply:
Agreed. There is no need for a filter as all the tags in that docker image/repository are just git refs. If/when that changes, this will need updating to filter (though fluxv2 works differently anyway, and might be a higher priority.